experiments/modern/src/App.js

```diff
@@ -133,15 +133,10 @@ function Button({ id, image, action, x, y, downImage, tooltip, node, children })
     return null;
   }
 
-  const hooks = node.js_getActiveHooks();
   const eventHandlers = {};
-  if (hooks.includes("onLeftClick")) {
-    eventHandlers["onClick"] = e => {
-      if (hooks.includes("onLeftClick")) {
-        node.js_trigger("onLeftClick");
-      }
-    };
-  }
+  eventHandlers["onClick"] = e => {
```

Comment:
I left this like this instead of just putting it directly on the node because I assume we will have some utility functions that buildup common bits of event handling/dispatching for shared events across node types....maybe just a long excuse for not tidying things